.github/fixtures/new-fixture-template/cliff.toml

```diff
@@ -1,12 +1,11 @@
 [changelog]
-# changelog header
+# A static header for the changelog.
 header = """
 # Changelog\n
 All notable changes to this project will be documented in this file.\n
 """
-# template for the changelog body
-# https://keats.github.io/tera/docs/#introduction
-body = """
+# A Tera template to be rendered for each release in the changelog (see https://keats.github.io/tera/docs/#introduction).
+body_template = """
```

Comment:
can't we make `header` a template too and remove `_template` from all the changelog parts?

Reply:
Yes, that would certainly be a good idea. I didn't do that yet, because I wanted to keep the changes distinct. But I can add it to this PR.